src/runtime-rs/crates/runtimes/virt_container/src/container_manager/container.rs

```diff
@@ -387,6 +388,9 @@ fn amend_spec(spec: &mut oci::Spec) -> Result<()> {
             resource.network = None;
         }
 
+        *sandbox_pidns = handle_pid_namespace(&linux.namespaces);
```

Comment:
This function needs a unit test, but see https://github.com/kata-containers/kata-containers/pull/4692#issuecomment-1189955702